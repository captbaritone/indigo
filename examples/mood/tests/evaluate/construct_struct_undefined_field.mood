struct Foo {
  a: i32,
  b: i32
}

pub fn test(): i32 {
    Foo { a: 1_i32, b: 1_i32, c: 1_i32 };
    0_i32
}