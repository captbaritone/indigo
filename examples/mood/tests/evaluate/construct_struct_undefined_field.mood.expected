Error: Undefined struct field: "c".:
 --> construct_struct_undefined_field.mood:7:31

  | pub fn test(): i32 {
7 |     Foo { a: 1_i32, b: 1_i32, c: 1_i32 };
  |                               ^ "c" is not a field of "Foo".
  |     0_i32
